.gitignore

```diff
@@ -5,3 +5,7 @@ resources
 .netlify
 TODO
 .DS_Store
+
+# JetBrains IDE
+.idea/
```

Comment:
Nice. Maybe we should add a bunch of common IDE dot files that we should ignore. But not in this PR.